docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.lists.scss

```diff
@@ -98,6 +114,14 @@
     @include hb-ext-icon-color('black');
     background-image: none;
 
+    
+    @include hb-themes(('colorful', 'airy')) {
+      .hb-dark-pattern &,
+      .hb-dark-inversion .views-element-container & {
+        @include hb-pairing-color('border-color', 'tertiary-highlight-darken-10');
```

Comment:
Can we make this `border-color: transparent;` instead? On a light background the border provides a nice hover state. Since we are now on a dark background, the border is no longer necessary.

You will need to add the border: transparent for the dark inversion and dark pattern for the hover state as well to override the default border-color on line 128 below.

```
      .hb-dark-pattern &,
      .hb-dark-inversion .views-element-container & {
        border-color: transparent;
      }
```

Reply:
This is what the hover state looks like in light mode:
http://g.recordit.co/us1ONjIMWh.gif

If I add the border transparent to the focus state, the border on hover state is removed completely, which I'm fine with, I just want to clarify this is what we want.
http://g.recordit.co/UNPCJKN0n3.gif


----- 

Confirmed this via Slack conversation with Heather, just leaving that here for confirmation that we decided to go with the hover state featured here.